Magento_Theme/templates/root.phtml

```diff
@@ -1,69 +1,101 @@
 <!doctype html>
 <html <?= $htmlAttributes ?>>
     <head <?= $headAttributes ?>>
-        <link
-            href="<?= /* @noEscape */ $this->getViewFileUrl('css/styles.css') ?>"
-            rel="stylesheet preload"
-            as="style"
-        />
-
         <link
             rel="preload"
-            href="<?= /* @noEscape */ $this->getViewFileUrl('fonts/work-sans-regular.woff2') ?>"
+            href="<?= $this->getViewFileUrl('fonts/work-sans-regular.woff2') ?>"
             as="font"
             type="font/woff2"
             crossorigin
         >
 
         <link
             rel="preload"
-            href="<?= /* @noEscape */ $this->getViewFileUrl('fonts/work-sans-medium.woff2') ?>"
+            href="<?= $this->getViewFileUrl('fonts/work-sans-medium.woff2') ?>"
             as="font"
             type="font/woff2"
             crossorigin
         >
 
         <link
             rel="preload"
-            href="<?= /* @noEscape */ $this->getViewFileUrl('fonts/work-sans-bold.woff2') ?>"
+            href="<?= $this->getViewFileUrl('fonts/work-sans-bold.woff2') ?>"
             as="font"
             type="font/woff2"
             crossorigin
         >
 
         <link
             rel="preload"
-            href="<?= /* @noEscape */ $this->getViewFileUrl('fonts/playfair-display-regular.woff2') ?>"
```

Comment:
1. We don't have any tools that check it
2. The output is safe, there is no way to get unpredictable results, from function that always get the same props, because they are static.